nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -132,6 +141,9 @@ If this is checked, then the binaries will be built and pushed to chacra even if
       - nfs-ganesha-debian
 
     builders:
+      - inject:
+          properties-content: |
+            CEPH_BUILD={scenario}
```

Comment:
I think because you've used it as the default value for ``CEPH_BRANCH`` you won't need this. You'll just need to use ``CEPH_BRANCH`` in your build scripts.